### ISSUE TITLE
Problem Exectuting on CloudML

### ISSUE THREAD

**Issue by csitaula** [rstudio/keras#341]
I could not execute the following code successfully on the CloudML although it works locally. I wanted to store the data on my machine and it did not work. I migrated my data to the bucket address and tried to run, still it is raising an error that it can't recognize the path. Any one please help me to execute my code on the CloudML? 
library(keras)
library(cloudml)
# create the base pre-trained model
base_model <- application_vgg16(weights = 'imagenet', include_top = FALSE)

# model <- keras_model_sequential()%>%
#   base_model%>%
#   layer_flatten() %>%
#   layer_dense(units = 256,activation = 'relu') %>%
#   layer_dense(units = 7,activation = 'sigmoid')
# add our custom layers
predictions <- base_model$output %>% 
  layer_global_average_pooling_2d() %>% 
  layer_dense(units = 1024, activation = 'relu') %>% 
  layer_dense(units = 7, activation = 'softmax')

# this is the model we will train
model <- keras_model(inputs = base_model$input, outputs = predictions)

# first: train only the top layers (which were randomly initialized)
# i.e. freeze all convolutional InceptionV3 layers
freeze_weights(base_model)

# compile the model (should be done *after* setting layers to non-trainable)
model %>% compile(optimizer = 'rmsprop', loss = 'categorical_crossentropy')
train_datagen = image_data_generator(
  rescale = 1/255,
  rotation_range = 40,
  width_shift_range = 0.2,
  height_shift_range = 0.2,
  shear_range = 0.2,
  zoom_range = 0.2,
  horizontal_flip = TRUE,
  fill_mode = "nearest"
)
test_datagen <- image_data_generator(rescale = 1/255)
train_generator <- flow_images_from_directory(
  gs_local_dir("gs://nyu-training-testing/NYU/train"),
  train_datagen,
  target_size = c(224, 224),
  batch_size = 20,
  class_mode = "categorical"
)
test_generator <- flow_images_from_directory(
  gs_local_dir("gs://nyu-training-testing/NYU/test"),
  test_datagen,
  target_size = c(224, 224),
  batch_size = 20,
  class_mode = "categorical"
)

model %>% compile(
  loss = "categorical_crossentropy",
  optimizer = optimizer_rmsprop(lr = 2e-5),
  metrics = c("accuracy")
  
)
model%>% fit_generator(
  train_generator,
  steps_per_epoch = 3,
  epochs = 3,
  validation_data = test_generator,
  validation_steps = 5
)

**Comment by jjallaire** [comment]
It would be helpful to see the text of the error you are receiving. Also, if you could capture this in an intermediate variable and print it that would be helpful: `gs_local_dir("gs://nyu-training-testing/NYU/test")`

**Comment by csitaula** [comment]
Yes, this is the error. I loaded data in the cloud using this directory, but it showed an error on this command.  I have data loaded on my hard drive, how to load it along with the script without uploading it on the cloud? I'm in trouble.
